### PR TITLE
Feature/1077 add more logging

### PR DESCRIPTION
**Pull request by ckr123**

### CODE REVIEW

File: domains/certificates/Query.API/API/DataSyncSyncer/DataSyncService.cs [hunk 1]
```diff
@@ -48,6 +48,12 @@ public DataSyncService(IDataSyncClient client, ILogger<DataSyncService> logger,
                     masterData.MeteringPointOwner,
                     cancellationToken
                 );
+                logger.LogInformation(
+                    "Successfully fetched {numberOfMeasurements} measurements in period from {from} to: {to}",
```
Comment (duizer) [inline domains/certificates/Query.API/API/DataSyncSyncer/DataSyncService.cs:52]: Please add GSRN here. That is useful when we have more than 1 metering point.

File: domains/certificates/Query.API/API/DataSyncSyncer/DataSyncService.cs [hunk 2]
```diff
@@ -48,6 +48,12 @@ public DataSyncService(IDataSyncClient client, ILogger<DataSyncService> logger,
...
+                    result.Count,
+                        DateTimeOffset.FromUnixTimeSeconds(dateFrom).ToString("o"),
```
Comment (duizer) [inline domains/certificates/Query.API/API/DataSyncSyncer/DataSyncService.cs:54]: Can this be indented on the same level as the other arguments?

**Review by duizer — CHANGES_REQUESTED** [review]

File: domains/certificates/Query.API/API/DataSyncSyncer/DataSyncService.cs (hunk 1 above)
Comment (ckr123) [reply domains/certificates/Query.API/API/DataSyncSyncer/DataSyncService.cs]: Fixed in e961e28

File: domains/certificates/Query.API/API/DataSyncSyncer/DataSyncService.cs (hunk 2 above)
Comment (ckr123) [reply domains/certificates/Query.API/API/DataSyncSyncer/DataSyncService.cs]: fixed in e961e28

**Review by duizer — APPROVED** [review]

**Pull request merged**